mypy/test/data.py

```diff
@@ -101,6 +101,13 @@ def parse_test_cases(
                     src_path = join(os.path.dirname(path), arg)
                     with open(src_path) as f:
                         files.append((join(base_path, 'typing.pyi'), f.read()))
+                elif p[i].id == 'add-module':
```

Comment:
This whole block is not necessary, just put the "pseudo stub" for ``attrs`` in ``mypy/test-data/unit/lib-stub``.

Reply:
Ok cool. I didn't know if that was ok since it's not a stdlib.